app/views/events/show.html.erb

```diff
@@ -32,12 +32,12 @@
           <div style="float:left;margin-top: 10px;">
             <meta itemprop="startDate" content="<%= l @event.starttime, format: '%Y-%m-%dT%H:%M' %>">
             <b>
-              <%= l @event.starttime, format: :from_long_date_time %>
+              <%= l @event.starttime, format: @event.all_day? ? :from_long_date : :from_long_date_time %>

```

Comment:
maybe we want to create a method for that in the Event class? What do you think?